README.md

```diff
@@ -1,82 +1,200 @@
-# terraform-google-migrate
+<!-----
+ Copyright 2018 Google LLC
 
-This module was generated from [terraform-google-module-template](https://github.com/terraform-google-modules/terraform-google-module-template/), which by default generates a module that simply creates a GCS bucket. As the module develops, this README should be updated.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
 
-The resources/services/activations/deletions that this module will create/trigger are:
+      http://www.apache.org/licenses/LICENSE-2.0
 
-- Create a GCS bucket with the provided name
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+----->
 
-## Usage
 
-Basic usage of this module is as follows:
+# CFT Migrate for Compute Engine (Migrate for Compute Engine)
+
+Using Cloud Foundation Toolkit (CFT) you can quickly deploy a landing zone for your [Migrate for Compute Engine](https://cloud.google.com/migrate/compute-engine/) migration to Google Cloud Platform (GCP) projects, save time and avoid misconfiguration issues.
+
+
+## Migrate for Compute Engine Prerequisites
+
+
+### Migration Qualification
+
+Before proceeding with migration, you should already know:
+
+1.  How many approximately VMs will be migrated
+2.  What is migration target VMs source - Azure, AWS or on-prem
+3.  How many GCP projects you will need for migrated VMs
+4.  Do you have/plan on-prem or other cloud connection to GCP via [VPN](https://cloud.google.com/vpn/docs/concepts/overview) / [Interconnect](https://cloud.google.com/hybrid-connectivity/)
+5.  Is there any network load balancing involved
+6.  How much storage will be needed in GCP
+
+
+### Deployment Type
+
+There are two types of deployment that you can choose from using CFT for Migrate for Compute Engine:
+
+**Single-project** deployment where Migrate for Compute Engine frontend, Cloud Extension and migrated VMs hosted under the same GCP project:
```

Comment:
FYi I recommend we drop this entirely in a future version or at least move it to a branch/submodule.

Reply:
Single project deployment is currently a submodule and has an example.